packages/react-core/src/components/WizardComposable/examples/WizardBasic.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { WizardComposable, WizardComposableStep } from '@patternfly/react-core';
+
+export const WizardBasic: React.FunctionComponent = () => (
+  <WizardComposable height={400}>
+    <WizardComposableStep name="First step" id="first-step">
```

Comment:
This applies to all WizardComposableStep components in all three examples, but they just need to have unique IDs across examples rather than each having a step with id "first-step", etc.

Reply:
Are you suggesting that in some cases we make the examples use a different naming convention for the unique ids?

Reply:
yeah when you reuse id's in examples, then we have duplicate ids in the docs. so maybe the composable steps in the basic example can have ids like "basic-first-step"